locales/ta_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -9,34 +9,35 @@ msgid ""
 msgstr ""
 "Project-Id-Version: DuckDuckGo-Translation-0.000\n"
 "POT-Creation-Date: 2019-01-16 23:42-0500\n"
-"PO-Revision-Date: 2019-01-16 23:49-0500\n"
+"PO-Revision-Date: 2022-03-26 12:58+0300\n"
 "Last-Translator: Community\n"
 "Language-Team: DuckDuckGo Community <community@duckduckgo.com>\n"
-"Language: இந்தியாவில் தமிழ் (Tamil in India)\n"
+"Language: ta_IN\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Poedit 3.0.1\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr ""
+msgstr "!Bang தேடல் குறுக்குவழிகள்"
 
 #. Heading for the table column indicating what percent of the population is fully vaccinated
 msgctxt "Covid 19 module"
 msgid "% Fully Vaccinated"
-msgstr ""
+msgstr "%முழுமையாக தடுப்பூசி போட்டுக்கொண்டேன்"
```

Comment:
Should there be a whitespace between the `%` sign and the rest of the string? Maybe it's not the convention in your language, just checking. Please check the next string, too. Thanks!

Reply:
Yes that's a mistake from my side, I'm sorry for that. I have corrected that